code/_core/turf/simulated/hazard/_hazard.dm

```diff
@@ -1,5 +1,19 @@
 /turf/simulated/hazard/
```

Comment:
I think this can be coded differently to avoid having to add extra variables to turfs but instead adding its own effect that is tracked by the fishing rod. I can do that myself if you're not up to it.

Reply:
i tried in the past but im not good enough at this to make it work yet so if you could check it out i'd appreciate it